commit c670569bdd2e7154f5d8cb0583f69f4fbad2a1ba
Author: Cameron Gutman <aicommander@gmail.com>
Date:   Thu Oct 9 22:52:57 2025 -0500

    Manually specify the shaderc_combined library
    
    The Vulkan SDK for Windows doesn't include CMake or pkg-config files
    that would allow Meson to locate this itself.

diff --git a/src/glsl/meson.build b/src/glsl/meson.build
index b1f4c8d0..a7dc3f3e 100644
--- a/src/glsl/meson.build
+++ b/src/glsl/meson.build
@@ -1,5 +1,5 @@
 # shaderc
-shaderc = dependency('shaderc', version: '>=2019.1', required: get_option('shaderc'))
+shaderc = cxx.find_library('shaderc_combined', required: get_option('shaderc'))
 components.set('shaderc', shaderc.found())
 if shaderc.found()
   build_deps += shaderc
